### PR TITLE
refactor: switch to upstream openapi

### PR DESCRIPTION
**Pull request by skyd0me**
unleash proxy is still using https://github.com/Unleash/express-openapi, which is the forked and archived version of https://github.com/wesleytodd/express-openapi

As all required changes are already pushed to the upstream project, we can migrate to the upstream project.
This was already done for the unleash server (see https://github.com/Unleash/unleash/pull/5259) a while back but not for unleash proxy.

It also updates the lib to the latest version so that security patches can be applied (which we need to do very soon due to path-to-regexp etc.). See https://github.com/Unleash/unleash/pull/8170/files

### CODE REVIEW

**Review by AdrienFromToulouse — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     // "isolatedModules": true,                     /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
 
     /* Strict Type-Checking Options */
-    "strict": true,                                 /* Enable all strict type-checking options. */
+    // "strict": true,                                 /* Enable all strict type-checking options. */
```
Comment (AdrienFromToulouse) [inline tsconfig.json:29]: what is the reason to switch this config off?

**Review by chriswk — CHANGES_REQUESTED** [review]
Please readd the strict type-checking.

File: tsconfig.json (hunk 1 above)
Comment (skyd0me) [reply tsconfig.json]: the main reason was the lack of types for the lib. But I adjusted the type definitions we already had

**Comment by chriswk** [comment]
Hi @skyd0me - I appreciate this contribution, we've now done it ourselves with #197, will make sure to credit you.

**Pull request closed without merging**